ml-proto/test/resizing.wast

```diff
@@ -20,32 +14,26 @@
     (func $store_at_page_size (result i32) (i32.store (i32.const 0x10000) (i32.const 3)))
 
     (export "grow" $grow)
-    (func $grow (param $sz i32) (result i32)
-      (grow_memory (call $round_up_to_page (get_local $sz)))
-    )
-
-    (export "size_at_least" $size_at_least)
-    (func $size_at_least (param i32) (result i32) (i32.ge_u (memory_size) (get_local 0)))
+    (func $grow (param $sz i32) (result i32) (grow_memory (get_local $sz)))
```

Comment:
@kripken It looks like https://github.com/WebAssembly/design/pull/598 has not been merged which updated AstSemantics. People seemed to support changing `grow_memory` to accept units of pages rather than bytes and there seems to be some support for also returning the new size in pages.